ledger/sandbox-common/src/test/lib/scala/platform/sandbox/cli/CommonCliSpecBase.scala

```diff
@@ -6,21 +6,20 @@ package com.daml.platform.sandbox.cli
 import java.io.File
 import java.net.InetSocketAddress
 import java.nio.file.{Files, Paths}
-import java.time.Duration
-
 import com.daml.bazeltools.BazelRunfiles.rlocation
 import com.daml.ledger.api.tls.TlsConfiguration
 import com.daml.ledger.participant.state.v1
 import com.daml.ledger.test.ModelTestDar
-import com.daml.platform.configuration.MetricsReporter
-import com.daml.platform.configuration.MetricsReporter.{Graphite, Prometheus}
+import com.daml.metrics.MetricsReporter
+import com.daml.metrics.MetricsReporter.{Graphite, Prometheus}
 import com.daml.platform.sandbox.cli.CommonCliSpecBase._
 import com.daml.platform.sandbox.config.SandboxConfig
 import com.daml.platform.services.time.TimeProviderType
 import com.daml.ports.Port
 import org.scalatest.Assertion
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AnyWordSpec
+import scala.concurrent.duration._
```

Comment:
```suggestion
import scala.concurrent.duration.DurationInt
```

Not necessary but I usually prefer to not bring in scope things I don't use, mostly to prevent unwanted implicit conversions to be brought in scope.